commentHandler_test.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"os"
+	"testing"
+)
+
+var envVarOptions = []struct {
+	title          string
+	envName        string
+	envConfigVal   string
+	envExpectedVal string
+}{
+	{
+		title:          "envvar correctly set",
+		envName:        "maintainers_file",
+		envConfigVal:   "DEREK",
+		envExpectedVal: "DEREK",
+	},
+	{
+		title:          "Misspelt envVar Name",
+		envName:        "maintainers_fill",
+		envConfigVal:   "DEREK",
+		envExpectedVal: "MAINTAINERS",
+	},
+	{
+		title:          "envVar doesnt exist",
+		envName:        "",
+		envConfigVal:   "",
+		envExpectedVal: "MAINTAINERS",
+	},
+}
+
+func Test_getEnv(t *testing.T) {
+
+	for _, test := range envVarOptions {
+		t.Run(test.title, func(t *testing.T) {
+
+			os.Setenv(test.envName, test.envConfigVal)
+
+			envvar := getEnv("maintainers_file", "MAINTAINERS")
+
+			if envvar != test.envExpectedVal {
+				t.Errorf("Maintainers File - wanted: %s, found %s", test.envExpectedVal, envvar)
+			}
+			os.Unsetenv(test.envName)
+		})
+	}
+}
+
+var actionOptions = []struct {
+	title          string
+	body           string
+	expectedAction string
+}{
+	{
+		title:          "Correct reopen command",
+		body:           "Derek reopen",
+		expectedAction: "reopen",
+	},
+	{ //this case replaces Test_Parsing_Close
+		title:          "Correct close command",
+		body:           "Derek close",
+		expectedAction: "close",
+	},
+	{
+		title:          "invalid command",
+		body:           "Derek dance",
+		expectedAction: "",
+	},
+	{
+		title:          "Longer reopen command",
+		body:           "Derek reopen: ",
+		expectedAction: "reopen",
+	},
+	{
+		title:          "Longer close command",
+		body:           "Derek close: ",
```

Comment:
Is this new?

Reply:
Its always been there as it drops through the pre-existing `isValidCommand` condition.